swift-sdk/IterableConstants.h

```diff
@@ -24,3 +22,9 @@ typedef void (^OnSuccessHandler)(NSDictionary * _Nullable data);
  The prototype for the completion handler block that gets called when an Iterable call fails
  */
 typedef void (^OnFailureHandler)(NSString * _Nullable reason, NSData *_Nullable data);
+
+/* Swift equivalents (for when ready to move to 100% Swift */
```

Comment:
Great info. 
Maybe we should add space after `//`? So 
`// typealias ...` etc.